modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/io/Clients.scala

```diff
@@ -69,6 +70,24 @@ object Clients {
       .withRequestTimeout(readTimeout)
       .withMaxTotalConnections(maxConnections)
       .resource
+      .map(Retry[F](retryPolicy, redactHeadersWhen))
+
+  private def retryPolicy[F[_]] =
+    RetryPolicy[F](
+      backoff,
+      retriable = {
+        case (_, Left(Command.EOF)) => true
```

Comment:
Please can make sure there is a github issue that describes this change, and then link to it with a code comment?  This is exactly the kind of code that looks very strange in 5 years time when we've forgotten why we did it.